impeller/compiler/spirv_sksl.cc

```diff
@@ -19,7 +19,7 @@ std::string CompilerSkSL::compile() {
   }
 
   options.es = false;
-  options.version = 300;
+  options.version = 100;
```

Comment:
@chinmaygarde I believe these are the correct settings for SkSL. Does this look right to you?